squire_web/src/tournament/players/mod.rs

```diff
@@ -2,7 +2,7 @@ use squire_sdk::{
     client::state::ClientState,
     model::{identifiers::RoundIdentifier, rounds::RoundStatus},
     players::PlayerId,
-    tournaments::TournamentId,
+    tournaments::{Tournament, TournamentId}, model::{identifiers::{PlayerIdentifier, TypeId}, rounds::RoundId},
```

Comment:
Super minor, but you don't need `TypeId` here. All of the id types (`RoundId`, `PlayerId, etc.) are specific types of TypeId. In fact, if you look at the definition of, say, `RoundId`, this is what you'll see:
```rust 
pub type RoundId = TypeId<Round>;
```
`TypeId` is a zero-overhead way of differentiating between a `Uuid` used to identify a player and one used to identify a round, but there should be little reason for you to use them directly. You should always use the pre-typed version of whatever you need.